docs/user_guide/forensics_dashboard.md

```diff
@@ -20,6 +20,12 @@ The dashboard also helps with completing a forensics dashboard regarding a secur
 * They help determine what the problem is and where the problem has occurred.  
 * It contains valuable information that provides more insight about the matter such as: destination, file extensions, file names, file paths, IP addresses, sourcetypes, and much more.
 
+### Notable Event Assignment and Status
+* User can assign the notable events to user for the investigation.
+* User can update the status of Notable event based the investigation phase.
```

Comment:
* User can also update the status of the notable event based on the investigation phase.